examples/utils.py

```diff
@@ -1,9 +1,22 @@
 import numpy as np
+import pinocchio as pin
+import example_robot_data
 import os
 
 CURRENT_DIRECTORY = os.getcwd()
 DEFAULT_SAVE_DIR = CURRENT_DIRECTORY + '/tmp'
 
+def loadTalos():
+    robotComplete = example_robot_data.load("talos")
+    qComplete = robotComplete.model.referenceConfigurations["half_sitting"]
+
+    locked_joints = [20,21,22,23,28,29,30,31]
```

Comment:
Here I would propose to write the joints names instead of id, t make for a more robust and easy to understand code
```python
locked_joints_names = ["right_wrist", ...]
locked_joints_id = [robotComplete.getJointId(joint) for joint in locked_joints_names]
```